apps/cms.c

```diff
@@ -106,6 +106,7 @@ const OPTIONS cms_options[] = {
      "Generate a signed receipt for a message"},
     {"verify_receipt", OPT_VERIFY_RECEIPT, '<',
      "Verify receipts; exit if receipt signatures do not verify"},
+    {"digest", OPT_DIGEST, 's', "Sign a pre-computed digest"},
```

Comment:
please add something like: 

given in hex notation

Reply:
BTW, would be good if, alternatively, a file could be given that contains the digest value (in direct binary form).

Reply:
IMO this is not a requirement for this PR and can be added any time later.

Reply:
Well, I don't insist, and your example contains good hints how to make indirect use of an intermediate file.
How convenient/feasible this is depends on the use cases that people may have, where I don't have an overview.
For instance, not all scenarios (e.g., on Windows) will have the option to easily obtain CLI input from a file.
Just note that also for password input there are several options, including referencing a file containing it.
What do others think?

Reply:
Update: just saw @t8m's response - I agree that this can be added later.